frontend/src/views/DifferentialExpression/components/Main/components/Filters/components/FilterDropdown/style.ts

```diff
@@ -88,3 +88,16 @@ export const StyledAutocomplete = styled(
     order: -2;
   }
 `;
+
+export const StyledCallout = styled(Callout)`
+  width: 100%;
+  background-color: ${gray100};
+  ${fontBodyS}
+  font-weight: 400;
```

Comment:
the font weight is inherited from the default SDS font weights here, so I believe that's `regular`!

https://github.com/chanzuckerberg/sci-components/blob/96a2af14747a5413c1bbd45ad45c35a10d52f76d/packages/components/src/core/styles/common/SDSAppTheme.ts#L11

```suggestion
  font-weight: ${fontWeightRegular};
```